crates/maybenot/src/constants.rs

```diff
@@ -9,7 +9,7 @@ pub const VERSION: u8 = 2;
 pub const MAX_DECOMPRESSED_SIZE: usize = 1 << 20;
 
 /// The number of [`Event`](crate::event)s in the framework.
-pub const EVENT_NUM: usize = 12;
+pub const EVENT_NUM: usize = 13;
```

Comment:
In general, the library has two consumers: integrators and machine developers. The constants are primarily helpful for the latter. I looked over the constants and found an old one that can now be removed (yay!), but otherwise, it's worth keeping accessible.

For events, `Event::num_events` would make sense, but many other constants are hard to find similarly good homes for, so we'll be stuck with a constants module.

Reply:
That's what I get for writing too soon in the thread! Removed two more unnecessary constants and limited visibility of one to the crate. I still think it's tricky to find a good home for the other (useful) constants though.